CONTRIBUTING.md

```diff
@@ -325,7 +325,7 @@ Note that for VSCode, to enable "live" linting of TypeScript (and other) file ty
   "eslint.autoFixOnSave": true,
 ```
 
-It is **not** recommended to use `prettier` plugin on Kibana project. Because settings are in `eslintrc.js` file and it is applied to too many files that shouldn't be prettier-ized. 
+:warning: It is **not** recommended to use the [`Prettier` extension/IDE plugin](https://prettier.io/) while maintaining the Kibana project. Formatting and styling roles are set in the multiple `.eslintrc.js` files across the project and some of them use the [NPM version of Prettier](https://www.npmjs.com/package/prettier). Using the IDE extension might cause conflicts, applying the formatting to too many files that shouldn't be prettier-ized.
```

Comment:
Since I actually DO have Prettier enabled, and I've not noticed a problem, AND I'm _pretty sure_ I've seen **valid** vscode errors labeled as coming from Prettier, that it's all that bad to have Prettier enabled.  Also, especially since I don't believe I've run into a case where Prettier did it's thing when it shouldn't have.

I think it's worth keeping this in as a note though, since this seems to be common lore at this point, and I've not done any kind of exhaustive test on it.  Definitely good to have some rationale as to why this is a warning, which we didn't have before.

Reply:
@pmuellr Thank you for your input, your experience with the plugin is very helpful. I've added the *valid* errors (yet Prettier highlighting them as wrong) to the line.

Regarding your comment about the `:warning:`. My intention was to bring the attention by using this emoji ⚠️ . Do you think we should use another one instead? Here's a very complete list with all the possibilities: https://gist.github.com/rxaviers/7360908

Reply:
In some part of the code, I had linting issues between what prettier generates and what eslint expects. I think it's not *everywhere*, but regarding anonymous functions, prettier and eslint had different spacing rules in some folders:

eslint:
```
const a = function () {}
```

prettier
```
const a = function() {}
```

So I actually had to change my intellij watcher from prettier to eslint